commit 1fe0f7817e3eaa19103b70a09c4fb9d9837fbb93
Author: Stefan Walter <stefw@src.gnome.org>
Date:   Wed Feb 11 15:00:38 2009 +0000

    Add G_BEGIN_DECLS/G_END_DECLS so the library can be used with a C++
    compiler.
    
    svn path=/trunk/; revision=1554

diff --git a/library/gnome-keyring-memory.h b/library/gnome-keyring-memory.h
index babf958..64f988b 100644
--- a/library/gnome-keyring-memory.h
+++ b/library/gnome-keyring-memory.h
@@ -26,6 +26,8 @@
 
 #include <glib.h>
 
+G_BEGIN_DECLS
+
 /**
  * gnome-keyring-memory:Short_Description:
  * 
@@ -59,4 +61,6 @@ gboolean  gnome_keyring_memory_is_secure      (gpointer p);
 
 gchar*    gnome_keyring_memory_strdup         (const gchar* str);
 
+G_END_DECLS
+
 #endif /* GNOME_KEYRING_MEMORY_H */
diff --git a/library/gnome-keyring.h b/library/gnome-keyring.h
index 628bf8e..dc19a4c 100644
--- a/library/gnome-keyring.h
+++ b/library/gnome-keyring.h
@@ -29,6 +29,8 @@
 
 #include "gnome-keyring-result.h"
 
+G_BEGIN_DECLS
+
 #define GNOME_KEYRING_SESSION   "session"
 #define GNOME_KEYRING_DEFAULT   NULL
 
@@ -516,4 +518,6 @@ GnomeKeyringResult    gnome_keyring_daemon_set_display_sync         (const char
 
 GnomeKeyringResult    gnome_keyring_daemon_prepare_environment_sync (void);
 
+G_END_DECLS
+
 #endif /* GNOME_KEYRING_H */
